source/blog/2017-09-09-ember-2-16-released.md

```diff
@@ -0,0 +1,158 @@
+release of the framework.
+
+Consider using the
+[ember-cli-deprecation-workflow](https://github.com/mixonic/ember-cli-deprecation-workflow)
+addon if you would like to upgrade your application without immediately addressing
+deprecations.
+
+Two new deprecations are introduces in Ember.js 2.16:
+
+* TODO
+* TODO
+
+For more details on changes in Ember.js 2.16, please review the
+[Ember.js 2.16.0 release page](https://github.com/emberjs/ember.js/releases/tag/v2.16.0).
+
+### Upcoming Changes in Ember.js 2.17
+
+Ember.js 2.17 will introduce two new features:
+
+* TODO
+* TODO
+
+#### Deprecations in Ember.js 2.17
+
+Two new deprecations are introduces in Ember.js 2.17:
+
+* TODO
+* TODO
+
+For more details on the upcoming changes in Ember.js 2.17, please review the
+[Ember.js 2.17.0-beta.1 release page](https://github.com/emberjs/ember.js/releases/tag/v2.17.0-beta.1).
+
+---
+
+## Ember Data
+
+Ember Data is the official data persistence library for Ember.js applications.
+
+### Changes in Ember Data 2.16
+
+Ember Data 2.16 is a bugfix release. It contains no new features but
+it does have several bugfixes for [build warnings](https://github.com/emberjs/data/pull/5196),
+[header parsing](https://github.com/emberjs/data/pull/5029) and
+[outdated dependecies](https://github.com/emberjs/data/pull/5044).
+
+#### Deprecations in Ember Data 2.16
+
+There are no new deprecations introduced in Ember Data 2.16.
+
+For more details on changes in Ember Data 2.16, please review the
+[Ember Data 2.16.0 release page](https://github.com/emberjs/data/releases/tag/v2.16.0).
+
+
+### Upcoming changes in Ember Data 2.17
+
+Due to a late start in the Ember Data 2.16 beta cycle the Ember Data
+2.17.0-beta.1 release is currently the same code as 2.16.0. The beta
+releases will continue to receive beta bugfixes and it is expected to
+diverge from Ember Data 2.16.0 by the time 2.17.0 is released in 6
+weeks.
+
+#### Deprecations in Ember Data 2.17
+
+There are no planned deprecations introduced in Ember Data 2.17.
+
+For more details on the upcoming changes in Ember Data 2.17, please review the
+[Ember Data 2.17.0-beta.1 release page](https://github.com/emberjs/data/releases/tag/v2.17.0-beta.1).
+
+---
+
+## Ember CLI
+
+Ember CLI is the command line interface for managing and packaging Ember.js
+applications.
+
+### Upgrading Ember CLI
+
+You may upgrade Ember CLI separately from Ember.js and Ember Data! To upgrade
+your projects using `yarn` run:
+
+```
+yarn upgrade ember-cli
+```
+
+To upgrade your projects using `npm` run:
+
+```
+npm install --save-dev ember-cli
+```
+
+After running the
+upgrade command run `ember init` inside of the project directory to apply the
+blueprint changes. You can preview those changes for [applications](https://github.com/ember-cli/ember-new-output/compare/v2.15.0...v2.16.0)
+and [addons](https://github.com/ember-cli/ember-addon-output/compare/v2.15.0...v2.16.0).
+
+### Changes in Ember CLI 2.16
```

Comment:
- project config is cached now for faster builds (https://github.com/ember-cli/ember-cli/pull/7333, https://github.com/ember-cli/ember-cli/pull/7309, https://github.com/ember-cli/ember-cli/pull/7270)

- Fix ember install for scoped packages (https://github.com/ember-cli/ember-cli/pull/7109)

- Bundler refactorings

Reply:
if they get merged in time:

- Update "ember-cli-uglify" to v2.0.0

- Use "ci" mode for testem.js